src/renderer/pages/Onboarding/index.tsx

```diff
@@ -13,6 +13,11 @@ export default function Onboarding() {
   const [backupFolders, setBackupFolders] = useState<BackupFolder[]>([]);
   const [slideIndex, setSlideIndex] = useState<number>(0);
   const [backupsModalOpen, setBackupsModalOpen] = useState(false);
+  const [platform, setPlatform] = useState<string>('');
```

Comment:
ℹ️  There is a hook to get the platform `useClientPlatform`

Reply:
glorious, thx didn't see it